scripts/call_contracts.py

```diff
@@ -51,7 +52,13 @@ def cli(
 
     account = account_from_keyfile(keystore_file, password)
     web3 = make_web3(eth_rpc, account)
-    contracts = contracts_for_web3(web3, artifacts_dir, abi_dir)
+
+    abi_manager = ABIManager(abi_dir)
+    deployment = beamer.artifacts.load(artifacts_dir, ChainId(web3.eth.chain_id))
+
+    contracts = {}
+    for name in ("RequestManager", "FillManager", "MintableToken"):
```

Comment:
Just an idea here, how about using get_erc20_abi function to get token abi and getting token_address as input then we can use this freely with mainnet deployments as well

Reply:
but can be done in seperate pr